jekyll/index.markdown

```diff
@@ -422,30 +421,6 @@ This feature is supported by the [Type Hierarchy Supertypes LSP request](https:/
 
 We created [an issue](https://github.com/microsoft/language-server-protocol/issues/1984) to seek clarification from the LSP maintainers. We will adjust this feature's design and behavior based on their response and your feedback.
 
-### Guessed Types
```

Comment:
Since this is now shipped, let's remove it from the `Design & Roadmap` but leave it here?